src/pages/chapter/application.tsx

```diff
@@ -1,9 +1,46 @@
-import { Flex } from "@chakra-ui/react";
+import { Button, Flex, HStack, Link, VStack } from "@chakra-ui/react";
+import React from "react";
+import { updateChapterProject } from "src/actions/Project";
+import ApplicationCard from "src/components/ApplicationCard";
+import NonprofitInfoCard from "src/components/NonprofitInfoCard";
+import { showError, showInfo } from "src/utils/notifications";
+import { ChapterProjectUpdate, ProjectStage } from "src/utils/types";
 
 function ChapterApplicationPage() {
+  // Todo: pass in the current project's id
+  const projectId = "616ba502a1667f4a0eadcfa1";
+  const nonprofitId = "60f208139c188f6f2fc71ffd";
+
+  const handleClaimProject = async () => {
+    const projectUpdate: ChapterProjectUpdate = {
+      status: ProjectStage.SCHEDULE_INTERVIEW,
+    };
+
+    try {
+      await updateChapterProject(projectId, projectUpdate);
+      showInfo("Successfully claimed project.");
+    } catch (e) {
+      const error = e as Error;
+      showError(error.message);
+    }
+  };
+
   return (
-    <Flex height="100%" width="100%">
-      <Flex margin="auto">View Application</Flex>
+    <Flex height="100%" width="100%" marginTop={10}>
+      <VStack margin="auto">
+        <HStack spacing={1125}>
```

Comment:
This is a good example of why absolute units can be a problem.  As you make the browser width smaller, this will prevent all content from being compressed because this spacing is constant.  A better thing to do here is to make this HStack a Flex component, and then use justifyContent="space-between" to make both elements as far apart as possible from each other.  That way, it does not prevent the content from reformatting for smaller screen sizes.